plugins/mediaPlayer/mediaPlayer.ts

```diff
@@ -85,6 +90,19 @@ export const initMediaPlayer = (
     activeMedia.value.registerEvents();
   }
 
+  function play() {
+    if (activeMedia.value) {
+      activeMedia.value.play();
+    } else {
+      initCurrentTrack();
+    }
+  }
+
+  function restart() {
+    if (queue.value.length === 1) play();
```

Comment:
I doubt that this actually has worked, because it doesn't reset the ended-flag of the track. I've refactored this in a new PR where I'm also implementing the single-repeat.